src/main/java/spoon/support/compiler/jdt/ReferenceBuilder.java

```diff
@@ -358,8 +360,14 @@ CtReference getDeclaringReferenceFromImports(char[] expectedName) {
 						if (packageBinding == null || packageBinding instanceof ProblemPackageBinding) {
 							// Big crisis here. We are already in noclasspath mode but JDT doesn't support always
 							// creation of a package in this mode. So, if we are in this brace, we make the job of JDT...
-							packageBinding = new PackageBinding(chars, null, environment, environment.module);
-						}
+							packageBinding = new PackageBinding(chars, null, environment, environment.module) {
+								@Override
+								public PlainPackageBinding getIncarnation(ModuleBinding arg0) {
+									//https://github.com/eclipse/eclipse.jdt.core/blob/master/org.eclipse.jdt.core/compiler/org/eclipse/jdt/internal/compiler/lookup/PlainPackageBinding.java#L43
```

Comment:
#3314 just as a link

Reply:
I don't understand, what do you mean?

Reply:
the class `PackageBinding ` was changed to an abstract class. To fix the problem, that you cant create an instance of it directly, i created an anonymous class. But the class needs to override the `public PlainPackageBinding getIncarnation(ModuleBinding arg0)` Method. I linked the original method from `PlainPackageBindung` to explain why this method always returns null.